uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/LoadLabUserTestRule.java

```diff
@@ -87,8 +95,23 @@ public void evaluate() throws Throwable {
         };
     }
 
+    private void createLabClient() {
+        final LabApiAuthenticationClient authenticationClient = new LabApiAuthenticationClient(
+                BuildConfig.LAB_CLIENT_SECRET
+        );
+        mLabClient = new LabClient(authenticationClient);
+    }
+
     public String getLabUserUpn() {
         return upn;
     }
 
+    public LabAccount getLabAccount() {
+        return mLabAccount;
+    }
+
+    public LabClient getLabClient() {
```

Comment:
Is this needed? Where is this used?

Reply:
There is a testcase that creates a second test account in Broker I believe. Figured giving access to client is better than creating a new one.

Reply:
Ok, that's fine then.